temporal/api/cloud/namespace/v1/message.proto

```diff
@@ -0,0 +1,93 @@
+syntax = "proto3";
+
+package temporal.api.cloud.namespace.v1;
+
+option go_package = "go.temporal.io/api/cloud/namespace/v1;namespace";
+
+import "temporal/api/cloud/sink/v1/message.proto";
+
+import "google/protobuf/timestamp.proto";
+
+message CertificateFilterSpec {
+    // the common_name in the certificate
+    string common_name = 1;
+    // the organization in the certificate
+    string organization = 2;
+    // the organizational_unit in the certificate
+    string organizational_unit = 3;
+    // the subject_alternative_name in the certificate
+    string subject_alternative_name = 4;
+}
+
+message CodecServerPropertySpec {
+    // server endpoints
+    string endpoint = 1;
+    // whether to pass access token, i.e. jwt
+    bool pass_access_token = 2;
+    // whether to include credentials
+    bool include_credentials = 3;
+}
+
+message NamespaceSpec {
+    // the region where the namespace is (or will be) active.
+    string region = 1;
+    // the base64 encoded ca cert(s) that the clients can use for authentication and authorization
+    string accepted_client_ca = 2;
+    // the number of days the workflows data will be retained for
+    int32 retention_days = 3;
+    // the custom search attributes to use for the namespace
+    // the name of the attribute is the key and the type is the value
+    // supported attribute types: text, keyword, int, double, bool, datetime, keyword_list
+    map<string, string> search_attributes = 4;
+    // certificate filters which, if specified, only allow connections from client certificates
+    // whose distinguished name properties match at least one of the filters.
+    repeated CertificateFilterSpec certificate_filters = 5;
+    // environment of the namespace. [optional]
+    // NOTE: currently there is no additional SLA or functional guarantee implied by the value of this field.
+    // supported environments: dev, test, prod
+    string environment = 6;
+    // codec server property spec needed for user to set and retrieve [optional]
+    CodecServerPropertySpec codec_spec = 7;
+    // the regions where the namespace is (or will be) located [optional]
+    repeated string passive_regions = 8;
+    // the export sink specifications keyed on the sink name [optional]
+    map<string, temporal.api.cloud.sink.v1.ExportSinkSpec> export_sinks = 9;
```

Comment:
For my particular use case, it would be nice to be able to reference a common export sink config since we use the same config globally across all namespaces (as history export is a platform-managed feature here).

Reply:
@robzienert thanks for the this feedback. 
We are tracking this as a feature as part of the "hierarchy" story where various configurations like ca-certs, search-attrs, export configs can trickle down a hierarchy sub-tree as defaults...
cc: @jlacefie @sergeybykov

Reply:
TY - Rob.  We released the first Export per Namespace with the understanding that some Namespace owners would like isolation at the S3 bucket. In a subsequent release we can provide Account level affordances for a default.